webrender/src/renderer.rs

```diff
@@ -1907,6 +1907,16 @@ impl Renderer {
                                     ExternalImageSource::RawData(data) => {
                                         self.device.update_pbo_data(&data[offset as usize..]);
                                     }
+                                    ExternalImageSource::Invalid => {
+                                        // Create a local buffer to fill the pbo.
+                                        let bpp = texture.get_bpp();
+                                        let width = stride.unwrap_or(rect.size.width * bpp);
```

Comment:
Sorry, I made a mistake for the image width.
The stride already contains the bpp.